mibidata/tiff.py

```diff
@@ -279,11 +279,13 @@ def _page_description(page):
 def _page_metadata(page, description):
     """Parses the page metadata into a dictionary."""
     assert page.tags['resolution_unit'].value == 3
-    assert page.tags['x_resolution'].value[0] ==\
-        page.tags['y_resolution'].value[0],\
+    x_resolution = page.tags['x_resolution'].value[0] / \
+        page.tags['x_resolution'].value[1]
+    y_resolution = page.tags['y_resolution'].value[0] / \
+                   page.tags['y_resolution'].value[1]
```

Comment:
jaylint here: The line above this has one tab indent for the line continuation and this one is lined up with the = sign.

Reply:
fixed

Reply:
Oh, I liked the other way better 😛